ecs-cli/Godeps/Godeps.json

```diff
@@ -516,7 +513,7 @@
 		},
 		{
 			"ImportPath": "golang.org/x/sys/windows",
-			"Rev": "478fcf54317e52ab69f40bb4c7a1520288d7f7ea"
+			"Rev": "99f16d856c9836c42d24e7ab64ea72916925fa97"
```

Comment:
So, after some digging, it would appear that the changes in `99f16d856c9836c42d24e7ab64ea72916925fa97` for `golang.org/x/sys/windows` are already reflected in the `/vendor` directory, but the revision SHA wasn't updated in the manifest. Checking out that version of the package and comparing it against what we have in our code, it looks like there are no real differences:
https://gist.github.com/SoManyHs/736f38ee56f818aeefa596dbf3791a7c